README.md

```diff
@@ -1,56 +1,41 @@
-# custom-action-template
-
-This template can be used to quickly start a new custom js action repository.  Click the `Use this template` button at the top to get started.
-
-## TODOs
-- Readme
-  - [ ] Update the Inputs section with the correct action inputs
-  - [ ] Update the Example section with the correct usage   
-- package.json
-  - [ ] Update the `name` with the new action value
-- main.js
-  - [ ] Implement your custom action
-- action.yml
-  - [ ] Fill in the correct name, description, inputs and outputs
-- check-for-unstaged-changes.sh
-  - [ ] If you encounter a permission denied error when the build.yml workflow runs, execute the following command: `git update-index --chmod=+x ./check-for-unstaged-changes.sh`
-- .prettierrc.json
-  - [ ] Update any preferences you might have
-- CODEOWNERS
-  - [ ] Update as appropriate
-- Repository Settings
-  - [ ] On the *Options* tab check the box to *Automatically delete head branches*
-  - [ ] On the *Options* tab update the repository's visibility
-  - [ ] On the *Branches* tab add a branch protection rule
-    - [ ] Check *Require pull request reviews before merging*
-    - [ ] Check *Dismiss stale pull request approvals when new commits are pushed*
-    - [ ] Check *Require review from Code Owners*
-    - [ ] Check *Include Administrators*
-  - [ ] On the *Manage Access* tab add the appropriate groups
+# Open PagerDuty Maintenance Window
+
+This action will open a PagerDuty Maintenance Window for the specified service(s) for the specified amount of time.  It returns the window ID as an output so the window can be closed prior to the expiration if desired.
 
 ## Inputs
-| Parameter | Is Required | Description           |
-| ----------|-------------|-----------------------|
-| `input-1` | true        | Description goes here |
-| `input-2` | false       | Description goes here |
+| Parameter           | Is Required | Description                                                    |
+| ------------------- | ----------- | -------------------------------------------------------------- |
+| `pagerduty-api-key` | true        | The PagerDuty api key that allows access to your services.     |
+| `service-ids`       | true        | The number of minutes to open the window for.  Defaults to 20. |
+| `description`       | false       | A description of the maintenance window. Defaults to empty.    |
+| `minutes`           | false       | A string array of PagerDuty Service IDs.                       |
```

Comment:
This doesn't seem like the right description.

Reply:
Fixed, I moved the service-ids and minutes args before so the required args were together and I for some reason didn't take their descriptions with them.